### PR TITLE
Removing flags and data related to Ethereum testnets

### PR DESCRIPTION
**Pull request by mrsmkl**

### CODE REVIEW

File: consensus/istanbul/backend/engine_test.go [hunk 1]
```diff
@@ -238,7 +238,7 @@ func getGenesisAndKeys(n int, isFullChain bool) (*core.Genesis, []*ecdsa.Private
 	}
 
 	// generate genesis block
-	genesis := core.DefaultOttomanGenesisBlock()
+	genesis := core.DefaultGenesisBlock()
```
Comment (kevjue) [inline consensus/istanbul/backend/engine_test.go:241]: There seems to be a really big different between the default ottoman genesis block's extra data and the default genesis block's.

Do you know what that difference is?

File: core/genesis_test.go [hunk 2]
```diff
@@ -35,9 +35,9 @@ func TestDefaultGenesisBlock(t *testing.T) {
 	if block.Hash() != params.MainnetGenesisHash {
 		t.Errorf("wrong mainnet genesis hash, got %v, want %v", block.Hash().Hex(), params.MainnetGenesisHash.Hex())
 	}
-	block = DefaultTestnetGenesisBlock().ToBlock(nil)
-	if block.Hash() != params.TestnetGenesisHash {
-		t.Errorf("wrong testnet genesis hash, got %v, want %v", block.Hash().Hex(), params.TestnetGenesisHash.Hex())
+	block = DefaultBaklavaGenesisBlock().ToBlock(nil)
```
Comment (kevjue) [inline core/genesis_test.go:38]: For the sake of completion, should a testcase for the alfajores genesis block be added here?

File: core/genesis_test.go [hunk 3]
```diff
@@ -95,14 +95,14 @@ func TestSetupGenesis(t *testing.T) {
 			wantConfig: customg.Config,
 		},
 		{
-			name: "custom block in DB, genesis == testnet",
+			name: "custom block in DB, genesis == baklava",
```
Comment (kevjue) [inline core/genesis_test.go:98]: For the sake of completion, should there be a test case for the alfajores genesis here?

File: cmd/clef/main.go [hunk 4]
```diff
@@ -95,7 +95,7 @@ var (
 	chainIdFlag = cli.Int64Flag{
 		Name:  "chainid",
 		Value: params.MainnetChainConfig.ChainID.Int64(),
-		Usage: "Chain id to use for signing (1=mainnet, 3=Ropsten, 4=Rinkeby, 5=Goerli)",
+		Usage: "Chain id to use for signing (42220=mainnet)",
```
Comment (kevjue) [inline cmd/clef/main.go:98]: Should we also add the other networks here? 

baklava=40120
alfajores= 44786

File: cmd/devp2p/discv4cmd.go [hunk 5]
```diff
@@ -181,7 +181,7 @@ func discv4Crawl(ctx *cli.Context) error {
 }
 
 func parseBootnodes(ctx *cli.Context) ([]*enode.Node, error) {
-	s := params.RinkebyBootnodes
+	s := params.AlfajoresBootnodes
```
Comment (kevjue) [inline cmd/devp2p/discv4cmd.go:184]: Is there a specific reason why you are using the Alfajores bootnodes here?

File: cmd/geth/chaincmd.go [hunk 6]
```diff
@@ -134,8 +134,6 @@ The export-preimages command export hash preimages to an RLP encoded stream`,
 			utils.CacheFlag,
 			utils.SyncModeFlag,
 			utils.FakePoWFlag,
```
Comment (kevjue) [inline cmd/geth/chaincmd.go:136]: Should we add in the alfajores and baklava testnet flags?

File: cmd/geth/chaincmd.go [hunk 7]
```diff
@@ -181,9 +179,6 @@ Use "ethereum dump 0" to dump the genesis block.`,
 			utils.DataDirFlag,
 			utils.AncientFlag,
 			utils.CacheFlag,
-			utils.TestnetFlag,
```
Comment (kevjue) [inline cmd/geth/chaincmd.go:184]: Should we add in the alfajores and baklava testnet flags?

File: mobile/geth.go [hunk 8]
```diff
@@ -177,10 +177,10 @@ func NewNode(datadir string, config *NodeConfig) (stack *Node, _ error) {
 			return nil, fmt.Errorf("invalid genesis spec: %v", err)
 		}
 		// If we have the testnet, hard code the chain configs too
-		if config.EthereumGenesis == TestnetGenesis() {
-			genesis.Config = params.TestnetChainConfig
+		if config.EthereumGenesis == AlfajoresGenesis() {
+			genesis.Config = params.AlfajoresChainConfig
 			if config.EthereumNetworkID == 1 {
-				config.EthereumNetworkID = 3
+				config.EthereumNetworkID = 40120
```
Comment (kevjue) [inline mobile/geth.go:183]: Should the baklava case be handled here as well?

File: params/config.go [hunk 9]
```diff
@@ -72,29 +63,6 @@ var (
...
-			common.HexToAddress("0x78d1aD571A1A09D60D9BBf25894b44e4C8859595"), // Martin
-			common.HexToAddress("0x286834935f4A8Cfb4FF4C77D5770C2775aE2b0E7"), // Zsolt
-			common.HexToAddress("0xb86e2B0Ab5A4B1373e40c51A7C712c70Ba2f9f8E"), // Gary
-			common.HexToAddress("0x0DF8fa387C602AE62559cC4aFa4972A7045d6707"), // Guillaume
-		},
-		Threshold: 2,
-	}
-
 	// TestnetChainConfig contains the chain parameters to run a node on the Ropsten test network.
 	TestnetChainConfig = &ChainConfig{
```
Comment (kevjue) [inline params/config.go:67]: Is this still needed?

File: consensus/istanbul/backend/engine_test.go (hunk 1 above)
Comment (mrsmkl) [reply consensus/istanbul/backend/engine_test.go:241]: Mostly it's the mainnet genesis validators. They are replaced below in `AppendValidatorsToGenesisBlock`

File: params/config.go (hunk 9 above)
Comment (mrsmkl) [reply params/config.go:76]: I think it's used in tests.

File: cmd/devp2p/discv4cmd.go (hunk 5 above)
Comment (mrsmkl) [reply cmd/devp2p/discv4cmd.go]: Looks like they just had some default here

File: params/config.go [hunk 10]
```diff
@@ -33,6 +33,14 @@ var (
 	BaklavaGenesisHash   = common.HexToHash("0x09bb180829b78343cc752e110c2ece86dad46904ffad9a5791b3440f140f1d7f")
 )
 
+var (
+	MainnetNetworkId   = uint64(42220)
```
Comment (kevjue) [inline params/config.go:37]: Nice!

**Pull request merged**